docker-compose.yml

```diff
@@ -1,12 +1,10 @@
 version: "3.8"
+
 services:
-  client:
+  webapp:
     stdin_open: true
     build:
       context: .
-      dockerfile: Dockerfile.dev
+      target: nextjs-dev-app
     ports:
-    - "3000:3000"
-    volumes:
-    - "/app/node_modules"
-    - "./:/app"
```

Comment:
@stuart-madetech I wasn't sure whether you were wanting us to stand up the application locally by running `docker-compose up` or whether we would be using `npm run start`?  I had assumed the later, which is why I had removed the volumes but will put them back in if we expect to run the application using Docker